packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilitySocketMappingStatus.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { Tooltip } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+import Fa from 'svelte-fa';
+import { router } from 'tinro';
+
+import type { DockerSocketMappingStatusInfo } from '/@api/docker-compatibility-info';
+
+import Label from '../../ui/Label.svelte';
+import ProviderInfoCircle from '../../ui/ProviderInfoCircle.svelte';
+
+let isMac = $state(false);
+let isWindows = $state(false);
+let isLinux = $state(false);
+
+let dockerSocketMappingStatusInfo: DockerSocketMappingStatusInfo | undefined = $state(undefined);
+
+let engineType: 'kubernetes' | 'podman' | 'docker' | undefined = $state(undefined);
+
+async function refreshSocketMappingStatus(): Promise<void> {
+  dockerSocketMappingStatusInfo = await window.getSystemDockerSocketMappingStatus();
+
+  if (dockerSocketMappingStatusInfo?.serverInfo?.type === 'podman') {
+    engineType = 'podman';
+  } else if (dockerSocketMappingStatusInfo?.serverInfo?.type === 'docker') {
+    engineType = 'docker';
+  } else {
+    engineType = undefined;
+  }
+}
+
+onMount(async () => {
+  const platform = await window.getOsPlatform();
+
+  isMac = platform === 'darwin';
+  isLinux = platform === 'linux';
+  isWindows = platform === 'win32';
+
+  // ask once to get the result
+  await refreshSocketMappingStatus();
+});
+</script>
+
+<div
+  class="bg-[var(--pd-invert-content-card-bg)] rounded-md mt-2 ml-2 divide-x divide-gray-900 flex flex-col lg:flex-row">
+  <div class="flex flex-row grow px-2 py-2 justify-between text-[color:var(--pd-invert-content-card-text)]">
+    <div class="flex flex-col">
+      <div class="flex flex-row items-center text-[color:var(--pd-invert-content-card-text)]">
+        System socket status
+
+        <div class="mx-2">
+          <Tooltip tip="Refresh the status">
+            <button
+              aria-label="refresh status"
+              class="text-xs text-violet-500"
```

Comment:
Violet is not in our palette, just there to avoid breakage but no new use. Should come from the color registry.

Reply:
it's how our refresh icons are displayed
https://github.com/containers/podman-desktop/blob/4681eecd2a536bcb9c4978f54815744f8d075a2a/packages/renderer/src/lib/preferences/PreferencesRenderingItem.svelte#L74

Reply:
what would be the color of the refresh icon @ekidneyrh for both light and dark theme ?
cc @ekidneyrh 
looks like it's purple-600

Reply:
solved by https://github.com/containers/podman-desktop/pull/9208